react/features/participants-pane/actions.native.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import { openDialog } from '../base/dialog';
+import { hideDialog, openDialog } from '../base/dialog';
```

Comment:
Since they are sheets, this got refactored into openSheet /closeSheet. Sorry I didn't get to your PR before that refactor.